website/docs/03_components/03_package-repository.md

```diff
@@ -24,7 +22,32 @@ A package repository must use the following directory structure to be fully comp
 
 The root `index.yaml` contains a list of all packages available from this repository. It is used primarily by client software to aid explorability.
 All files related to a package reside in a directory that must have the same name as the package. 
-Inside a package's directory there may be a `versions.yaml` that contains a list of all versions available for this package.
-If such a `versions.yaml` file exists, there must be a subdirectory for each version containing a `package.yaml` file.
-If no `versions.yaml` file exists, there must be a `package.yaml` in the package's directory.
+Inside a package's directory there must be a `versions.yaml` that contains a list of all versions available for this package.
+There must be a subdirectory for each version containing a `package.yaml` file.
 A `package.yaml` contains a manifest of that package which holds information such as longer descriptions and included files.
+
+### Version Numbers
```

Comment:
@pmig @kosmoz please have a look at this in particular – that's how I interpreted what we decided back then.